tensorflow/python/ops/random_ops.py

```diff
@@ -19,6 +19,7 @@
 from __future__ import print_function
 
 import numpy as np
+import tensorflow as tf
```

Comment:
You should follow the convention to only import what you are using. I don't think you should import the top level tensorflow module here

Reply:
Thanks @BryanCutler.